src/plugins/saved_objects_finder/public/finder/saved_object_finder.tsx

```diff
@@ -355,23 +360,35 @@ export class SavedObjectFinderUi extends React.Component<
           ]
         : undefined,
       toolsRight: this.props.children ? <>{this.props.children}</> : undefined,
+      toolsLeft: this.props.leftChildren ? <>{this.props.leftChildren}</> : undefined,
     };
 
     return (
-      <EuiInMemoryTable
-        loading={this.state.isFetchingItems}
-        itemId="id"
-        items={this.state.items}
-        columns={columns}
-        data-test-subj="savedObjectsFinderTable"
-        message={this.props.noItemsMessage}
-        search={search}
-        pagination={pagination}
-        sorting={sorting}
-        onTableChange={({ sort }) => {
-          this.setState({ sort });
-        }}
-      />
+      <EuiFlexGroup direction="column">
+        <EuiFlexItem>
+          {this.props.helpText ? (
+            <EuiText size="s" color="subdued">
+              {this.props.helpText}
+            </EuiText>
+          ) : undefined}
+        </EuiFlexItem>
```

Comment:
```suggestion
        {this.props.helpText ? (
          <EuiFlexItem>
            <EuiText size="s" color="subdued">
              {this.props.helpText}
            </EuiText>
          </EuiFlexItem>
        ) : undefined}
```
It looks like this change causes additional space above the finder for consumers who don't use `helpText`. I think we could solve it by either making the whole `EuiFlexItem` conditional, or by not adding the `helpText` prop and just manually rendering the text above the finder where it's used in cases -- I'd be fine with either approach.

Before:
<img width="681" alt="Screenshot 2023-03-09 at 1 04 40 PM" src="https://user-images.githubusercontent.com/25592674/224101653-2ab63408-2564-4934-a6e0-86225ca042b5.png">

After:
<img width="681" alt="Screenshot 2023-03-09 at 12 57 41 PM" src="https://user-images.githubusercontent.com/25592674/224101232-10a1b484-52af-4656-bed4-06a8f2d60f01.png">

Reply:
Good catch! Fixed